src/file.cr

```diff
@@ -237,23 +240,34 @@ class File < IO::FileDescriptor
     basename(path).chomp(suffix)
   end
 
+  private alias UserRep = System::User | String | UInt32 | Int32
```

Comment:
I'd rather avoid using private aliases like these, in docs they'll end up pretty cryptic for end-user. Also, the name is not easily understable (I guess `Rep` suffix stands for `Representation` but still, not everyone might grok that).

Reply:
I can replace them with the long form, but the lines become awfully long.

Reply:
IMO it's better than an unknown, cryptic alias.

Reply:
We could add 
```
alias System::User::Representation  = System::User | String | UInt32 | Int32
```